02-peer-protocol.md

```diff
@@ -954,6 +956,12 @@ There's little point offering spam updates: it implies a bug.
 
 The `num_htlcs` field is redundant, but makes the packet length check fully self-contained.
 
+The recommendation to require recent messages recognizes the reality
+that networks are unreliable: nodes might not realize their peers are
+offline until after sending `commitment_signed`.  Once
+`commitment_signed` is sent, the sender considers itself bound to
+those HTLCs, and cannot close incoming HTLCs.
```

Comment:
```suggestion
those HTLCs, and cannot instantly fail related incoming HTLCs anymore.
```

Reply:
Agree with @pm47, "close" for HTLC seems wrong/too undetailed here.  Both @pm47  and @rustyrussell rewordings seem fine.